src/runtime_config/default_actions.js

```diff
@@ -418,4 +421,49 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "transition_node_to_trusted",
+    new Action(
+      function (args) {
+        checkType(args.node_id, "string", "node_id");
+      },
+      function (args) {
+        const node = ccf.kv["public:ccf.gov.nodes.info"].get(
+          ccf.strToBuf(args.node_id)
+        );
+        if (node !== undefined) {
```

Comment:
I would remove this. It should be an error if the node doesn't exist.

Reply:
Ok